pytest_fixtures/authsource_fixtures.py

```diff
@@ -0,0 +1,207 @@
+"""Test class for Active Directory Feature
+
+:Requirement: Ldapauthsource
+
+:CaseAutomation: Automated
+
+:CaseLevel: Integration
+
+:CaseComponent: LDAP
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import copy
+from airgun.session import Session
+
+from nailgun import entities
+from robottelo.config import settings
+from robottelo.constants import AUDIENCE_MAPPER
+from robottelo.constants import GROUP_MEMBERSHIP_MAPPER
+from robottelo.constants import LDAP_ATTR
+from robottelo.constants import LDAP_SERVER_TYPE
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture
+from robottelo.rhsso_utils import create_mapper
+from robottelo.rhsso_utils import get_rhsso_client_id
+from robottelo.rhsso_utils import run_command
+
+
+@fixture(scope='module')
+def ldap_data():
```

Comment:
I wouldn't necessarily consider this or `ipa_data` to be fixtures, as there is no setup involved, even within the framework. Its just creating a data structure from configuration data - this is something that could be defined as a module variable and simply used in the actual fixtures that achieve some configuration.

Reply:
This is old code, thanks for raising. I agree on that no need makes as a fixture.